rotkehlchen/db/dbhandler.py

```diff
@@ -247,7 +247,7 @@ def get_settings(self) -> DBSettings:
             elif q[0] == 'ui_floating_precision':
                 settings['ui_floating_precision'] = int(q[1])
             elif q[0] == 'taxfree_after_period':
-                settings['taxfree_after_period'] = int(q[1])
+                settings['taxfree_after_period'] = int(q[1]) if q[1] else None
```

Comment:
Nicely spotted! Thanks a lot. Can you also add a changelog entry to the `changelog.rst` file? It should solve [this](https://github.com/rotkehlchenio/rotkehlchen/issues/197) bug and say something along the lines:

"Rotkehlchen no longer crashes at restart if a "No" tax_free_period is given"

Reply:
Yup. I'll get that added.